Assets/MixedRealityToolkit/Utilities/MixedRealityServiceRegistry.cs

```diff
@@ -0,0 +1,222 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit
+{
+    /// <summary>
+    /// Static class that represents the Mixed Reality Toolkit service registry.
+    /// </summary>
+    /// <remarks>
+    /// The service registry is used to enable discovery of and access to active Mixed Reality Toolkit services at
+    /// runtime without requiring direct code reference to a singleton style component.
+    /// </remarks>
+    public static class MixedRealityServiceRegistry
+    {
+        // todo: is this the data type we want?
```

Comment:
> // todo: is this the data type we want? [](start = 7, length = 40)

Nit: What's the resolution on this TODO?

Reply:
haha! forgot to remove that one :)